legacy/i18n/locales/en-US.json

```
@@ -3,7 +3,8 @@
   "menu.allWallets": "All Wallets",
   "menu.catalystVoting": "Catalyst Voting",
   "menu.settings": "Settings",
-  "menu.faq": "FAQ",
+  "menu.faqTitle": "Any Questions?",
```

I think we need to update the design.
```suggestion
  "menu.faqTitle": "Any question?",
```

---

I also believe that the text for the CTA is `Find answers on Yoroi Helpdesk`